test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -1187,6 +1188,49 @@ def forward(self, x):
         x = torch.randn(3, 4, 5, 6, 7)
         self.run_model_test(NegSlice(), train=False, input=(x,), batch_size=BATCH_SIZE, use_gpu=False)
 
+    def test_dynamicslice_3inputs_onnx2caffe2(self):
```

Comment:
I think https://github.com/pytorch/pytorch/blob/master/caffe2/python/onnx/tests/onnx_backend_test.py or https://github.com/pytorch/pytorch/blob/master/caffe2/python/onnx/tests/c2_ref_test.py is more appropriate place to add the following tests.